### PR TITLE
DDF-04702 Resolve 'createdBy is not defined' console error that occurs…

### PR DESCRIPTION
**Pull request by rymach**
… on result form creation

#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@AzGoalie

#### How should this be tested?
1. create result form
#### Any background context you want to provide?
None
#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4702

#### Screenshots
image

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,4 +37,5 @@ dependency-reduced-pom.xml
 buildNumber.properties
 .mvn/timing.properties
 coverage/
-.nyc_output
\ No newline at end of file
+.nyc_output
+STOPKEY
```
Comment (rymach) [inline .gitignore]: bonus

Comment (Lambeaux) [inline .gitignore]: ❓ Was this intentional? Do we really need to ignore `STOPKEY` I thought it was deployed as part of Solr and not living at the root of the repo?

**Comment by Lambeaux** [comment]
PR title is incomplete.

**Comment by vinamartin** [comment]
If the issue was introduced by an unreleased PR, you can give it the same ticket #.

File: .gitignore (hunk 1 above)
Comment (rymach) [reply .gitignore]: maybe it just pops it into whatever your working directory is when you start ddf?

Comment (rymach) [reply .gitignore]: I'm guessing most people start ddf from inside of the target directory which is already ignored

Comment (rymach) [reply .gitignore]: I removed this change

Comment (Lambeaux) [reply .gitignore]: 👍

**Comment by mojogitoverhere** [comment]
build now

**Comment by rymach** [comment]
People who liked this PR also liked: 
https://github.com/codice/ddf/pull/4697

**Comment by mojogitoverhere** [comment]
hero in progress

**Comment by mojogitoverhere** [comment]
## 🎉 Hero build was successful! 🎉 ##

I was able to successfully create new result forms

**Comment by mojogitoverhere** [comment]
build now

**Pull request merged**